Docs/ProjectSpec.md

```diff
@@ -782,7 +782,9 @@ A multiline script can be written using the various YAML multiline methods, for
 - [x] **name**: **String** - The name of the target
 - [ ] **parallelizable**: **Bool** - Whether to run tests in parallel. Defaults to false
 - [ ] **randomExecutionOrder**: **Bool** - Whether to run tests in a random order. Defaults to false
+- [ ] **useTestSelectionWhitelist**: **Bool** - Whether to use a list of selected tests. Defaults to nil.
```

Comment:
Would you ever set `useTestSelectionWhitelist` to false while providing `selectedTests`? I wonder if we can just surface `selectedTests` and if it's not empty internally set `useTestSelectionWhitelist` on `XCScheme.TestableReference` generation

Reply:
In case if `useTestSelectionWhitelist` is false or nil, `selectedTests` will be ignored. 
I thought to make a logic without using `useTestSelectionWhitelist`, but it seems too complicated and not so clear. For example, what if config provides both `skippedTests` and `selectedTests`, user can do this as well.